# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/debug/Debugger-setInstrumentation-bug1571169.js
# Commit: 27cea02bdee4
# Full Hash: 27cea02bdee4b9478bd89a07354a6eebd3d1d76b
# Author: Iain Ireland <iireland@mozilla.com>
# Date: 2021-06-01 09:43:14
# Regressor Bug: 1554524
# File Overlap Count: 1
# Description:
#   Bug 1624792: Remove instrumentation tests r=tcampbell
#   
#   This patch stack is based on the patches for bug 1554524 in more-or-less reverse order, with a few modifications so that the intermediate patches build. This is part 8.
#   
#   Depends on D116304
# ==============================================================================

diff -r 56f9a8184a88 -r 27cea02bdee4 js/src/jit-test/tests/debug/Debugger-setInstrumentation-bug1571169.js
--- a/js/src/jit-test/tests/debug/Debugger-setInstrumentation-bug1571169.js	Tue Jun 01 01:31:19 2021 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,18 +0,0 @@
-// |jit-test| error: Error
-
-var lfOffThreadGlobal = newGlobal();
-evaluate(`
-  nukeAllCCWs();
-  var g92 = newGlobal({ newCompartment: true });
-  var dbg = Debugger(g92);
-  var gdbg = dbg.addDebuggee(g92);
-  gdbg.setInstrumentation(
-    gdbg.makeDebuggeeValue((kind, script, offset) => {}),
-    ["breakpoint"]
-  );
-  gdbg.setInstrumentationActive(true);
-  g92.eval(\`
-    function basic() {}
-  \`);
-`);
-
